question/classes/local/bank/action_column_base.php

```diff
@@ -0,0 +1,95 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * A base class for actions that are an icon that lets you manipulate the question in some way.
+ *
+ * @package   core_question
+ * @copyright 2009 Tim Hunt
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace core_question\local\bank;
+
+defined('MOODLE_INTERNAL') || die();
+
+/**
+ * A base class for actions that are an icon that lets you manipulate the question in some way.
+ *
+ * @copyright 2009 Tim Hunt
+ * @author    2021 Safat Shahin <safatshahin@catalyst-au.net>
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+abstract class action_column_base extends column_base {
+
+    /**
+     * Title for this column. Not used if is_sortable returns an array.
+     */
+    protected function get_title(): string {
+        return '&#160;';
+    }
+
+    /**
+     * Get extra classes for the column.
+     *
+     * @return array any extra class names you would like applied to every cell in this column.
+     */
+    public function get_extra_classes():array {
+        return array('iconcol');
+    }
+
+    /**
+     * Print the icon of the action.
+     *
+     * @param $icon
+     * @param $title
+     * @param $url
+     */
+    protected function print_icon($icon, $title, $url): void {
+        global $OUTPUT;
+        echo '<a title="' . $title . '" href="' . $url . '">' . $OUTPUT->pix_icon($icon, $title) . '</a>';
```

Comment:
Replace the hard coded link with `html_writer::` or what the core $OUTPUT render provides